R/tbl_custom_summary.R

```diff
@@ -284,7 +282,16 @@ tbl_custom_summary <- function(
   tbl_custom_summary_inputs <- as.list(environment())
 
   # checking function inputs ---------------------------------------------------
-  # verify if checks about stat_fns should be added at this stage
+  if (rlang::is_function(stat_fns)) {
```

Comment:
This message allows inconsistent syntax to the other arguments in the package. I would prefer to keep the helpful error messaging we just added in broom.helpers. The user will still be prompted to update their syntax.

Reply:
It has been changed